infra/launch.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+import argparse
+import getpass
+import subprocess
+
+from infra import deploy
+from google.cloud import storage
+
+
+# Oddly enough, there's no API to simply fetch the current gcloud configuration...
+def gcloud_config():
+    client = storage.Client()
+    return {
+        "project": client.project,
+    }
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--tpu", type=str, required=True)
+    parser.add_argument("--project", type=str, default=gcloud_config()["project"])
+    parser.add_argument("--zone", type=str, required=True)
+    parser.add_argument("--docker_repository", type=str, required=True)
```

Comment:
would be nice if we could put this in a .config in the levanter base dir or something

Reply:
Ack, see above.